src/components/connection-map/ConnectionMap.tsx

```diff
@@ -14,9 +14,9 @@ import { feature } from 'topojson-client';
 import { Skeleton } from 'src/components/skeleton/Skeleton';
 import { theme } from 'src/styles/constants/theme';
 import type { BaseProps } from 'src/types/BaseProps';
-import { type CountryOption } from 'src/types/Country';
 import { type GeoJsonWorld } from 'src/types/GeoJsonWorld';
 import { getCountryCodeByName } from 'src/utils/getCountryCodeByName';
+import { type CountryOption } from 'src/utils/hooks/useCountryOptions';
```

Comment:
I think we should turn on the no-side-effect rule for amino, looks like we only use CountryOption type, but import the whole hook in. I doubt that it would increase the size significantly 🤔

Reply:
Hmm, can we do this in a separate PR?

Reply:
yah, just point it out, that's why I didn't request changes 😆